web/src/routes/(user)/trash/+page.svelte

```diff
@@ -0,0 +1,42 @@
+<script lang="ts">
+  import UserPageLayout from '$lib/components/layouts/user-page-layout.svelte';
+  import DeleteAssets from '$lib/components/photos-page/actions/delete-assets.svelte';
+  import RestoreAssets from '$lib/components/photos-page/actions/restore-assets.svelte';
+  import SelectAllAssets from '$lib/components/photos-page/actions/select-all-assets.svelte';
+  import AssetGrid from '$lib/components/photos-page/asset-grid.svelte';
+  import AssetSelectControlBar from '$lib/components/photos-page/asset-select-control-bar.svelte';
+  import EmptyPlaceholder from '$lib/components/shared-components/empty-placeholder.svelte';
+  import { AppRoute, AssetAction } from '$lib/constants';
+  import { createAssetInteractionStore } from '$lib/stores/asset-interaction.store';
+  import { AssetStore } from '$lib/stores/assets.store';
+  import { TimeBucketSize } from '@api';
+  import type { PageData } from './$types';
+  import { featureFlags } from '$lib/stores/server-config.store';
+  import { goto } from '$app/navigation';
+
+  export let data: PageData;
+
+  $: if ($featureFlags.loaded && !$featureFlags.recycleBin) {
+    goto(AppRoute.PHOTOS);
+  }
```

Comment:
```suggestion
  $: $featureFlags.trash || goto(AppRoute.PHOTOS);
```

Trash should start out as true (even when not loaded)